ast/import_test.go

```diff
@@ -63,6 +63,7 @@ func TestImportDeclaration_MarshalJSON(t *testing.T) {
                     "EndPos": {"Offset": 3, "Line": 2, "Column": 5}
                 }
             ],
+			"Aliases": null,
```

Comment:
Could you please test the case where the aliases are set? You can do that by adding some dummy/arbitrary aliases in the test declaration above (https://github.com/onflow/cadence/pull/3717/files#diff-5133e9d8f2ff8054c388461bfe306367de04a0ed58979bdcc84e3cd0463dd910R36) and asserting the serialization here

Reply:
Replaced the null case with the case where the aliases are set.